README.md

```diff
@@ -8,21 +8,21 @@ without sharing access to the source code of the project!
 
 ## Quickstart
 
-1. Install Broker: `TODO: add install script`
+1. Install Broker ([documentation](https://github.com/fossas/broker/blob/main/docs/README.md))
 2. Initialize Broker: `broker init`
-3. Configure the `.broker.yml` with your projects
+3. Configure the `config.yml` with your projects
```

Comment:
Should we say something like "You will get instructions on what to do and where config.yml is when you run `broker init`"?

Reply:
Done!